Drivers/Src/Contactors.c

```diff
@@ -103,7 +114,14 @@ ErrorStatus Contactors_Set(contactor_t contactor, bool state, bool blocking) {
 
     // change contactor to match state and make sure it worked
     setContactor(contactor, state);
-    bool ret = (bool)Contactors_Get(contactor);
+
+    // the motor contactor is the only contactor that controls has a sense pin for
+    if(blocking && contactor == MOTOR_CONTROLLER_CONTACTOR) {
```

Comment:
== takes precedence over && but i'd still add parentheses here:
```suggestion
    if(blocking && (contactor == MOTOR_CONTROLLER_CONTACTOR)) {
```